clusterloader2/pkg/measurement/interface.go

```diff
@@ -37,6 +37,8 @@ type MeasurementConfig struct {
 	// Identifier identifies this instance of measurement.
 	Identifier    string
 	CloudProvider string
+	// Overrides passed to the clusterloader2 via --testoverrides flag
+	Overrides map[string]interface{}
```

Comment:
I'm against this - especially that with the new TestSuite design we won't have global testoverrides.

Why do you need this?

Reply:
I'm not a fan of this either. All I'd like to do is to pass overrides to the measurement, so it can have conditions based on them. If you have another idea how to do it, especially with the new design, I'd be happy to change it.

Reply:
Hmm - actually I realized that maybe the only bad thing here is the comment :)
Because measurement is create per-test anyway - so we will just be passing different overrides in different tests.

I'm still not a a huge fan of it, but maybe actually it's not that bad.

So I guess I can live with it if you remove "via --testoverrides" flag", because this will soon disappear :)